lua/compe_ultisnips/init.lua

```diff
@@ -15,13 +41,17 @@ function M:determine(context)
 end
 
 function M:complete(args)
-  local received_snippets = vim.F.npcall(vim.call, 'UltiSnips#SnippetsInCurrentScope') or {}
+  local received_snippets = vim.F.npcall(vim.call, 'UltiSnips#SnippetsInCurrentScope', 1) or {}
```

Comment:
We should use `vim.g.current_ulti_dict_info` instead of it?

Reply:
I now remember why I didn't use it. I tried it when checking that the table was empty and it returned false, because it had some weird value. Something like [true]: 6.

That's why I left the first variable to check that it was empty, and then, using the global variable with the additional info.

Reply:
OK. I've understand.